packages/kbn-alerts-ui-shared/src/rule_form/rule_actions/rule_actions_system_actions_item.tsx

```diff
@@ -0,0 +1,274 @@
+          actionVariables: availableActionVariables,
+        })
+      );
+      validateActionParams(newParams);
+      onStoredActionParamsChange(storedActionParamsForAadToggle, newParams);
+    },
+    [
+      http,
+      action,
+      availableActionVariables,
+      dispatch,
+      validateActionParams,
+      onStoredActionParamsChange,
+      storedActionParamsForAadToggle,
+    ]
+  );
+
+  return (
+    <EuiAccordion
+      data-test-subj="ruleActionsSystemActionsItem"
+      initialIsOpen
+      borders="all"
+      style={{
+        backgroundColor: subdued,
+        borderRadius: euiTheme.border.radius.medium,
+      }}
+      id={action.id}
+      onToggle={setIsOpen}
+      buttonProps={{
+        style: {
+          width: '100%',
+        },
+      }}
+      arrowProps={{
+        css: css`
+          margin-left: ${euiTheme.size.m};
+        `,
+      }}
+      extraAction={
+        <EuiButtonIcon
+          data-test-subj="ruleActionsSystemActionsItemDeleteActionButton"
+          style={{
+            marginRight: euiTheme.size.l,
+          }}
+          aria-label={i18n.translate(
+            'alertsUIShared.ruleActionsSystemActionsItem.deleteActionAriaLabel',
+            {
+              defaultMessage: 'delete action',
+            }
+          )}
+          iconType="trash"
+          color="danger"
+          onClick={() => onDelete(action.uuid!)}
+        />
+      }
+      buttonContentClassName="eui-fullWidth"
+      buttonContent={
+        <EuiPanel
+          data-test-subj="ruleActionsSystemActionsItemAccordionButton"
+          color="subdued"
+          paddingSize="m"
+        >
+          <EuiFlexGroup alignItems="center">
+            <EuiFlexItem grow={false}>
+              {showActionGroupErrorIcon() ? (
+                <EuiToolTip content={ACTION_ERROR_TOOLTIP}>
+                  <EuiIcon
+                    data-test-subj="action-group-error-icon"
+                    type="warning"
+                    color="danger"
+                    size="l"
+                  />
+                </EuiToolTip>
+              ) : (
+                <Suspense fallback={null}>
+                  <EuiIcon size="l" type={actionTypeModel.iconClass} />
+                </Suspense>
+              )}
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiText>{connector.name}</EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiText size="s" color="subdued">
+                <strong>{actionType?.name}</strong>
+              </EuiText>
+            </EuiFlexItem>
+            {warning && !isOpen && (
+              <EuiFlexItem grow={false}>
+                <EuiBadge data-test-subj="warning-badge" iconType="warning" color="warning">
+                  {ACTION_WARNING_TITLE}
+                </EuiBadge>
+              </EuiFlexItem>
+            )}
+            {actionTypeModel.isExperimental && (
```

Comment:
I noticed that we did not provide the same functionality for normal actions. Should we add it?